host/4.0/buster/amd64/dotnet/dotnet-inproc/dotnet.Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Build the runtime from source
+ARG HOST_VERSION=4.0.0-preview.1.15799
```

Comment:
Note this isn't quite as clean as it could be, but works for now (this isn't an actual tag; I parse it below). Will follow up with improvements later.